### ISSUE TITLE
1.2.3 broke installation 

### ISSUE THREAD

**Issue by BarniRamirez** [feelfreelinux/octo4a#393]
yesterday the 1.2.2 was working on my device, today i decided to unistall and reinstall the app and now it says "installation corrupted". 
Is there a way to reinstall the old version?

**Comment by BarniRamirez** [comment]
I reinstalled the old .apk (octo4a-1.2.2.apk) "installing bootstrap (arm)" is ok, but when it gets to 
"downloading octoprint (newest)..." the app crashes and the "installation corrupted" message appears

**Comment by alufers** [comment]
What phone are you using?

**Comment by BarniRamirez** [comment]
I'm using a samsung galaxy tab 3 (SM-T310) with LineageOS-14.1, Android 7.1.2

**Comment by alufers** [comment]
Okay, I think it's an issue with 32-bit processors (armhf). I am currently testing a fix for this. A new release should be ready in a moment.

**Comment by BarniRamirez** [comment]
Okay ty! I'll try with the new version.

**Comment by BarniRamirez** [comment]
![Screenshot_20230325-160753](https://user-images.githubusercontent.com/92034607/227725755-9a565557-f46f-4b0e-a7e1-cd4922f872a3.png)
By the way this is the last thing it shows in the log right before it crashes ...

**Comment by alufers** [comment]
Okay, I've releases 1.2.4. Could you check it?

**Comment by BarniRamirez** [comment]
Okay now with the 1.2.4 it works again but (as with the 1.2.2) when i try to start the camera server the app crashes and tablet restarts.
However I solved the problem by keeping the camera permission in octoprint off and installing "ip webcam" (https://play.google.com/store/apps/details?id=com.pas.webcam&hl=it&gl=US) and running it simultaneously, then all I had to do was pasting the stream URL of the webcam app in the octoprint UI in settings > webcam & timelapse.

thank you for the support!
